docs/source/install/index.rst

```diff
@@ -54,6 +54,7 @@ Once it completes successfully, you will see the following output:
     Ubuntu 14.04 / 16.04 <deb>
     RHEL 7 / CentOS 7 <rhel7>
     RHEL 6 / CentOS 6 <rhel6>
+    Ansible Playbooks <ansible>
```

Comment:
@LindsayHill Sub-menu under "Installation" Is close to previous structure, see: https://docs.stackstorm.com/1.3/install/index.html
I'm for keeping it visible to get more users trying + testing this method for us, what could be especially beneficial considering we're running Ansible for own infra.